jfr-mappers/src/main/java/com/newrelic/jfr/toevent/EventToEvent.java

```diff
@@ -49,13 +48,4 @@ default boolean test(RecordedEvent event) {
   default Optional<Duration> getPollingDuration() {
     return Optional.empty();
   }
-
-  /**
-   * Returns the Java version where particular JFR events were added.
-   *
-   * @return int indicating Java version that introduced the event type
-   */
-  default int since() {
```

Comment:
I came across this method in three interfaces. I wasn't sure how to handle it with the backport to java 8 - do all implementations assume availability since java 8, even though support for 8 came chronologically after java 11? 

I looked for usages and couldn't find any, so I sidestepped the problem and got rid of the method thinking YAGNI.

Not sure this was the best approach, so I'm calling it out in case there's a reason to keep it around.